web/src/app/settings/suppliers/page.tsx

```diff
@@ -29,17 +36,22 @@ const SuppliersPage = () => {
 			<CustomTable
 				title="Suppliers"
 				columnSchema={columnSchema}
-				dataSource={dataSource}
+				dataSource={suppliers?.data ?? []}
+				totalCount={suppliers?.totalCount ?? 0}
```

Comment:
pa wrap ko ani og parent property `paginationProps`
```suggestion
				paginationProps={
				  totalCount={suppliers?.totalCount ?? 0}
    				  handleChangePage={handleChangePage}
    				  page={currentPage}
    				  rowsPerPage={limit}
    				  handleChangeRowsPerPage={handleChangeRowsPerPage}
				}
```